addons/website_slides/static/tests/activity_patch.test.js

```diff
@@ -48,18 +37,8 @@ test("refuse course access", async () => {
         request_partner_id: partnerId,
         res_model: "slide.channel",
     });
-    onRpc("action_refuse_access", (args) => {
-        expect(args.args).toHaveLength(1);
-        expect(args.args[0]).toHaveLength(1);
-        expect(args.args[0][0]).toBe(channelId);
-        expect(args.kwargs.partner_id).toBe(partnerId);
-        asyncStep("access_refused");
-        // random value returned in order for the mock server to know that this route is implemented.
-        return true;
-    });
     await start();
     await openFormView("slide.channel", channelId);
     await contains(".o-mail-Activity");
     await click("button", { text: "Refuse Access" });
-    await waitForSteps(["access_refused"]);
```

Comment:
The tests should have an extra step to verify whatever changed from server call.